client/src/component/AvailabilityWidget.js

```diff
@@ -104,95 +127,107 @@ const AvailabilityWidget = (props) => {
   const handleDayToggle = (event) => {
     setDays({ ...daysSelected, [event.target.value]: event.target.checked });
   };
-    return (
-        <Box className={classes.forOuterBox}>
-            <img alt="logo" src={logo}></img>
-            <Paper className={classes.forPaper}  elevation={3}>
-                <Box m={3}>
-                    <ProfileHeader percent={75} heading={props.heading} />
-                </Box>
-                <Divider/>
-                <Box m={3}>
-                    <Grid container alignItems="flex-start" direction="column">
-                        <Grid item>
-                            <Typography alight="left" variant="subtitle2">
-                                Available hours:
+  return (
+    <Box className={classes.forOuterBox}>
+      <img alt="logo" src={logo}></img>
+      <Paper className={classes.forPaper} elevation={3}>
+        <Box m={3}>
+          <ProfileHeader percent={75} heading={props.heading} />
+        </Box>
+        <Divider />
+        <Box m={3}>
+          <Grid container alignItems="flex-start" direction="column">
+            <Grid item>
+              <Typography alight="left" variant="subtitle2">
+                Available hours:
+              </Typography>
+            </Grid>
+            <Grid container alignItems="center">
+              <FormControl variant="outlined" className={classes.formControl}>
+                <Select
+                  labelId="start-time-label"
+                  id="start-time"
+                  value={startTime}
+                  onChange={handleChangeStart}
+                >
+                  {timeOptions.map((timeOptions) => (
+                    <MenuItem key={timeOptions} value={timeOptions}>
+                      {timeOptions}
+                    </MenuItem>
+                  ))}
+                </Select>
+              </FormControl>
+              <Typography alight="left" variant="h4">
+                -
+              </Typography>
+              <FormControl variant="outlined" className={classes.formControl}>
+                <InputLabel htmlFor="end-time"></InputLabel>
+                <Select
+                  labelId="end-time-label"
+                  id="end-time"
+                  value={endTime}
+                  onChange={handleChangeEnd}
+                >
+                  {timeOptions.map((timeOptions) => (
+                    <MenuItem key={timeOptions} value={timeOptions}>
+                      {timeOptions}
+                    </MenuItem>
+                  ))}

```

Comment:
When using map on an array with a plural name, each item is usually given the singular - i.e., `timeOptions.map(timeOption =>...`